### DISCUSSION TITLE
Reason why you need spring foraging/crops

### DISCUSSION THREAD

**Discussion by waynn** [nathan-alden-sr/stardew-valley#26] [General]
@nathan-alden-sr 

One of my runs ended because I didn't realize that the entire reason you need to have both of those is because you need two bundles before the Vault opens up. This might be something that advanced players just know, but I completely forgot it on one of my playthroughs.

**Comment by nathan-alden-sr** [comment]
See the last bullet point [here](https://github.com/nathan-alden-sr/stardew-valley/blob/master/min-max/1.5/year-1-spring-1.md#advice-and-warnings) for the forageable advice.

See the bullet point [here](https://github.com/nathan-alden-sr/stardew-valley/blob/master/min-max/1.5/year-1-spring-3.md#advice-and-warnings) for the Cauliflower advice.

I will add a similar warning about Potatoes later today.

**Reply by waynn** [reply]
Oh sorry, what I meant was I didn't realize you needed to complete two bundles before the Vault bundle opened up, and that's the entire reason you need to make sure you have those eight things. 

How I actually screwed up my run is I ate all my parsnips on a subsequent day (two or three days after harvesting) completely forgetting to save three. 😁 but that's just my own idiocy, I just thought I could still save the run by planting parsnips.

**Reply by nathan-alden-sr** [reply]
I'll add a note on Spring 1 that makes this super-obvious. I had early runs ended by the lack of context, as well.

**Reply by nathan-alden-sr** [reply]
https://github.com/nathan-alden-sr/stardew-valley/commit/a33f6b7ee6b4bf5f9427288344f2d76ffeb0999c
